libs/components/modals/testing/src/confirm/confirm-button-harness-filters.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseHarnessFilters } from '@angular/cdk/testing';
+
+/**
+ * A set of criteria that can be used to filter a list of SkyConfirmButtonHarness instances.
+ */
+export interface SkyConfirmButtonHarnessFilters
+  extends Omit<BaseHarnessFilters, 'selector'> {
+  /**
+   * Only find instances whose content matches the given value.
+   */
+  textContent?: string | RegExp;
```

Comment:
I'd go with `text` since it's just a button and wouldn't include complicated inner HTML. See the token harness filters as an example.